lte/gateway/c/oai/tasks/sgw/sgw_handlers.c

```diff
@@ -2368,8 +2371,213 @@ int sgw_handle_nw_initiated_actv_bearer_rsp(
     "Sending S5_NW_INIT_ACTIVATE_BEARER_RSP to PGW with EBI %d\n",
     act_ded_bearer_rsp->ebi);
   rc = itti_send_msg_to_task(TASK_PGW_APP, INSTANCE_DEFAULT, message_p);
+
+  OAILOG_FUNC_RETURN(LOG_SPGW_APP, rc);
+}
+
+/*
+ * Handle NW initiated Dedicated Bearer Deactivation from PGW
+ */
+uint32_t sgw_handle_nw_initiated_deactv_bearer_req(
+  const itti_s5_nw_init_deactv_bearer_request_t
+  *const itti_s5_deactiv_ded_bearer_req)
+{
+  MessageDef *message_p = NULL;
+  uint32_t rc = RETURNok;
+  uint32_t i = 0;
+
+  OAILOG_FUNC_IN(LOG_SPGW_APP);
+  OAILOG_INFO(LOG_SPGW_APP,
+    "Received Dedicated Bearer Req Dectivation from PGW for TEID %u\n",
```

Comment:
Be more consistent with log messages
maybe "Received nw_initiated_deactv_bearer_req from PGW for TEID %u\n",

Please check all log messages added in this PR

Reply:
done